### PR TITLE
Tdr 487 grafana ecs

### PR DESCRIPTION
**Pull request by TomJKing**

### CODE REVIEW

File: ecs/variables.tf [hunk 1]
```diff
@@ -1,14 +1,47 @@
...
+  default = ""
+}
+
 variable "file_format_build" {
   default = false
 }
 
+variable "file_system" {
+  default = {}
+}
```
Comment (MancunianSam) [inline ecs/variables.tf:29]: Are you using this file_system variable anywhere?

File: ecs/variables.tf [hunk 2]
```diff
@@ -1,14 +1,47 @@
...
+
+variable "grafana_build" {
+  default = false
+}
+
 variable "project" {}
+
+variable vpc_id {
+  default = ""
```
Comment (MancunianSam) [inline ecs/variables.tf:42]: And this vpc_id doesn't seem to be used either.

File: ecs/variables.tf (hunk 1 above)
Comment (TomJKing) [reply ecs/variables.tf]: Good point, removed.

File: ecs/variables.tf (hunk 2 above)
Comment (TomJKing) [reply ecs/variables.tf]: Removed

**Review by MancunianSam — APPROVED** [review]

**Pull request merged**